packages/atomic/cypress/integration/breadbox/breadbox-assertions.ts

```diff
@@ -191,9 +191,7 @@ export function assertDeselectColorFacet(index: number) {
 
```

Comment:
Some breadcrumb analytics assertions were used in facet tests, so I converted them to the new format. This new format will also wait until `TestFixture.interceptAliases.UA` if it wasn't already intercepted.